umake/__init__.py

```diff
@@ -85,6 +85,17 @@ def set_logging_from_args(args, parser):
         _setup_logging()
 
 
+def set_loading_frameworks(args):
+    """Choose if all frameworks are load ignoring any unknown sys.argv options"""
```

Comment:
I would change the comment to:
"""Set partial or complete framework loading condition based on arg"""